### ISSUE TITLE
Processo Judicial

### ISSUE THREAD

**Issue by franklinbaldo** [poliquin/brazilnum#12]
A numeração única de processos judiciais foi instituída pelo CNJ em 2006 e é muito relevante no meio jurídico. 
https://www.cnj.jus.br/programas-e-acoes/numeracao-unica/

**Comment by glasswalk3r** [comment]
I would like to give a shot to implement this, if this project is still being maintained. Please let me know @poliquin

**Comment by poliquin** [comment]
Yes, you're welcome to implement this. I haven't been actively working on this package because it mostly does what I need now, but pull requests are welcome.

**Comment by glasswalk3r** [comment]
> Yes, you're welcome to implement this. I haven't been actively working on this package because it mostly does what I need now, but pull requests are welcome.

~~First I created the PR #14 in order to have a foundation before adding new features.~~

**Comment by glasswalk3r** [comment]
@franklinbaldo , you want to take a look here: https://github.com/glasswalk3r/brazilian-ids
